ansible/molecule/quickstart/prepare.yml

```diff
@@ -0,0 +1,20 @@
+- name: Prepare
+  hosts: all
+
+  tasks:
+    - name: set english environment locale
+      import_tasks: ../utils/set-locale.yaml
+
+    - name: Install Git   # required by geerlingguy.nodejs
+      package:
+        name: git
+        state: present
+      become: yes
+
+    - name: install Node.js
+      include_role:
+        name: geerlingguy.nodejs
+        apply:
+          become: true
+      vars:
+        nodejs_version: 10.x
```

Comment:
@jslopez aquí no usamos normalmente nodejs 12.x ?

Reply:
de hecho node 10 ya esta end-of-life jajaja

igual el quickstart esta con node 10 (https://github.com/magnet-cl/django-project-template/blob/674467b8be7ab12f87f038ecc4998f82b45ae5cf/ansible/playbooks/roles/quickstart/tasks/node.yaml#L17)

debieramos subirlo a 12 o 14?

Reply:
https://github.com/magnet-cl/django-project-template/pull/99